docs/get-started/index.md

```diff
@@ -10,6 +10,7 @@ There are several ways to get your feet wet with LVGL. Here is one recommended o
 3. Read the [Quick overview](https://docs.lvgl.io/latest/en/html/get-started/quick-overview.html) page of the documentation (15 minutes)
 4. Set up a [Simulator](https://docs.lvgl.io/latest/en/html/get-started/pc-simulator.html) (10 minutes)
 5. Try out some [Examples](https://github.com/lvgl/lv_examples/)
+6. Try LVGL on ESP32 M5Stack device with [Tasmota and Berry](https://tasmota.github.io/docs/LVGL_in_10_minutes/) (10 minutes)
 6. Port LVGL to a board. See the [Porting](https://docs.lvgl.io/latest/en/html/porting/index.html) guide or check the ready to use [Projects](https://github.com/lvgl?q=lv_port_&type=&language=)
```

Comment:
I think it should be 7 now.

Reply:
As we do not mention another platform here either, it wouldn't be fair to mention only Tasmota. 

What about 
```suggestion
6. Check out the [Getting started](https://docs.lvgl.io/master/get-started/index.html) guides for platform-specific tutorials. 
```

Reply:
Thanks, that's fair indeed. However the link at 6. is now pointing to the same page, isnt't it? Or am I missing somehing?